Gemfile

```diff
@@ -4,12 +4,23 @@ source 'https://rubygems.org'
 
 gemspec
 
+group :development, :test do
+  gem 'pry'
+end
+
 group :development do
   gem 'rubocop'
 end
 
 group :test do
   gem 'bcrypt'
+  gem 'rspec-rails'
+  if defined?(JRUBY_VERSION)
+    gem 'activerecord-jdbcsqlite3-adapter'
```

Comment:
this seems more apt https://github.com/jruby/activerecord-jdbc-adapter

Reply:
Okay, Jruby seems to have a psych issue, I believe the fix is just installing psych before the tests...TBD

Reply:
...I can't replicate it locally

Reply:
advice appreciated 😓

Reply:
blowing up cache to see if that works

Reply:
I have no clue, but I agree on using the other gem.

Reply:
I tried using it, but it's still going haywire on bundle install